CenterForOpenScience/ember-osf-web
# [ENG-2157] Add headTags so that favicons changes for branded registries

- Ticket: [ENG-2157]
- Feature flag: n/a

## Purpose

Make sure favicons for branded services appear in the browser tabs.

## Summary of Changes

- Modified the `branded` route so that it will take the favicon asset link and put it in the head tags for all pages under the `branded` route.
- Modified the `overview` route to do the same thing.

## Side Effects

None.

## QA Notes
Make sure favicons appear on browser tabs for branded pages. Remember to add the favicons as assets to the provider in the admin app.


[ENG-2157]: https://openscience.atlassian.net/browse/ENG-2157
